src/Asset/Asset.php

```diff
@@ -65,7 +66,11 @@ final public function __construct()
      */
     public static function getDefinition(): AssetDefinition
     {
-        $definition = static::$definition ?? null;
+        // Get the definition without triggering PHPStan issue
+        // see https://github.com/phpstan/phpstan/issues/8808
+        // see https://github.com/phpstan/phpstan-src/pull/3019
```

Comment:
this has been rejected

Reply:
Not sure I can argue with the creator of PHPStan himself. I simplified the proposed fix in our side, to not use the Reflection API.